### PR TITLE
Lint

### PR DESCRIPTION
**Pull request by altaudio**

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -1,13 +1,17 @@
 {
   "parser": "babel-eslint",
-  "extends": ["@jokeyrhyme/es2015"],
+  "extends": ["eslint-config-airbnb"],
   "ecmaFeatures": {
   },
 
   "env": {
+    "browser": true,
+    "node": true
```
Comment (richardgill) [inline .eslintrc]: do we always have node: true turned on?

File: .eslintrc [hunk 2]
```diff
@@ -1,13 +1,17 @@
...
   },
 
-  "globals": {},
+  "globals": {
+    "document" : false
```
Comment (richardgill) [inline .eslintrc]: ? Wasn't expecting this?

File: .eslintrc [hunk 3]
```diff
@@ -1,13 +1,17 @@
...
+  },
 
   "rules": {
     semi: [2, "never"]
```
Comment (richardgill) [inline .eslintrc]: you're missing all our standard rules from react-seed?

File: package.json [hunk 4]
```diff
@@ -3,47 +3,89 @@
...
-    "lint_js": "eslint src/scripts",
-    "lint_sass": "node_modules/.bin/sass-lint -c sass-lint.yml -v",
-    "lint": "npm run lint_html && npm run lint_sass && npm run lint_js",
-    "test": "npm run lint",
-    "deploy": "npm run build && firebase deploy"
+    "upgrade": "yarn upgrade-interactive",
+    "deploy": "yarn run build && firebase deploy --token \"<TOKEN>\"",
+    "test": "yarn check && npm run lint",
+    "start": "webpack-dev-server",
+    "prod": "cross-env NODE_ENV=production webpack-dev-server --env.prod=true",
```
Comment (richardgill) [inline package.json]: prod, build and start should probably not come in yet. Bring the old webpack commands from the lhs back.

File: webpack.config.js [hunk 5]
```diff
@@ -16,11 +16,11 @@ module.exports = {
       {
         test: /\.js$/,
         exclude: /node_modules/,
-        loader: 'babel'
+        loader: 'babel-loader'
```
Comment (richardgill) [inline webpack.config.js]: not sure if you want these changes or not???

File: .eslintrc (hunk 2 above)
Comment (altaudio) [inline .eslintrc]: This was one of three options from this answer
https://stackoverflow.com/questions/41858052/solving-linter-error-no-undef-for-document

Comment (richardgill) [inline .eslintrc]: Does that not just mean to do either the env or the global but not both?

Comment (altaudio) [inline .eslintrc]: Have changed it just to global and that works

**Pull request closed without merging**